sdk/pbs/controller/api/resources/authmethods/auth_method_taggable_test.go

```diff
@@ -1,182 +0,0 @@
-package authmethods
-
-import (
-	"context"
-	"encoding/json"
-	"testing"
-	"time"
-
-	"github.com/hashicorp/boundary/sdk/pbs/controller/api"
-	"github.com/hashicorp/boundary/sdk/wrapper"
-	"github.com/hashicorp/eventlogger"
-	"github.com/hashicorp/eventlogger/filters/encrypt"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-	"google.golang.org/protobuf/types/known/structpb"
-	"google.golang.org/protobuf/types/known/wrapperspb"
-)
-
-func TestAuthMethod_Tags(t *testing.T) {
```

Comment:
I think we should be able to keep this test in place. I would potentially rename the file, but these tests should be asserting that the classification is correct, we should probably add similar tests for the other packages.

Reply:
I'm not sure what value these tests add to be honest. Classification is defined in the Protobuf files, anything downstream of that is automatically generated. These tests would catch us forgetting to run `gotags` on a file, or maybe adding a tag with a typo, but is that worth writing tests for? It feels like it would mostly test that `gotags` does the right thing.

Reply:
Discussed on zoom, we will keep these tests, but now that I'm playing around with them, I realize that they're going to be wrong until we add the classification anyway, so I'm going to punt on this until the next PR which adds the classification to the fields.